### ISSUE TITLE
Discard-like homotopy

### ISSUE THREAD

**Issue by calintat** [homotopy-io/homotopy-rs#599]
I just realised it is possible to have homotopies which in projection look like a 2-cell with one input and no outputs. I will call these "discard-like" homotopies. Currently we do not render the vertex for such homotopies, but I think we should.

For example:
![filename_todo(7)](https://user-images.githubusercontent.com/16837293/179812598-d8979168-e4c7-4062-a4a4-723e59968efd.svg) : ![filename_todo(5)](https://user-images.githubusercontent.com/16837293/179812609-3d6b01ab-c0fa-4215-9ef9-9ae0955f4d4a.svg) -> ![filename_todo(8)](https://user-images.githubusercontent.com/16837293/179812648-a56df2c6-4e61-404d-8408-ac7253710351.svg)

**Comment by jamievicary** [comment]
Agreed.

**Comment by calintat** [comment]
Similarly of course we can have "unit-like" homotopies.

**Comment by jamievicary** [comment]
I think this is a simple extra condition: if a "homotopy" has no inputs or no outputs, the vertex should be rendered.

**Comment by calintat** [comment]
Yes, I agree. In fact, I think the only homotopies we don't want to be rendered are identities (this includes braidings as they are locally identities if you restrict to the top layer).

In the labelled branch, we could have an extra condition to not render homotopies of orientation 0, so things like cups/caps.

**Comment by jamievicary** [comment]
Yes that makes sense to me.
